.github/ISSUE_TEMPLATE/app-request.yml

```
@@ -0,0 +1,50 @@
+---
+name: App request
+description: Submit an app
+labels: App Request
+body:
+  - type: input
+    id: app-name
+    attributes:
+      label: What is the name of the app?
+      placeholder: Ex - Zoom, AndroidBuddy
+    validations:
+      required: true
+      
+  - type: textarea
+    id: host
+    attributes:
+      label: (Optional) Where does the app hosted?
+      placeholder: Ex - Github link, Gitlab link, Sourceforge link
+    validations:
+      required: false
+  
+  - type: textarea
+    id: about
+    attributes:
+      label: About the app
+      description: What is its main purpose? How it works? 
+      placeholder: Tell us about the app here!
+    validations:
+      required: true
+      
+  - type: textarea
+    id: zip
+    attributes:
+      label: Upload file
+      description: "If you have a deb of the app, or if you've created a zip file for this app, or if you have any other relevant files, upload it here."
```

```suggestion
      description: If you have a deb of the app, or if you've created a zip file for this app, or if you have any other relevant files, upload it here.
```
The double-quotes appear in the rendered form.

---

This will display error as there is an `'` in the sentence, so we need to quote it.